public/index.html

```diff
@@ -9,7 +9,7 @@
     <meta property="og:image" content="<%= BASE_URL %>og-image-avalanche.png">
     <meta property="og:url" content="https://explorer.avax.network/">
     <meta property="og:type" content="website">    
-    <title>Avalanche Explorer: View All Activity on Avalanche</title>
+    <title>Avalanche Explorer</title>
```

Comment:
Should it be `View All Activities | Avalanche Explorer`?

Reply:
I don't think so -- the basic title gets replaced right away, and I feel like `View all activities` should just be used if you're on the overview page.